gateway/system/system.go

```diff
@@ -154,7 +154,14 @@ func NewSystem(opts *SystemOptions) (*System, error) {
 		dapiimpl.NewOtelTracingHandler(),
 		dapiimpl.NewUserAgentMetricsHandler(),
 		oapimetrics.NewStatsHandler(opts.Logger),
-	}, dataapiv1.StrictHTTPServerOptions{})
+	}, dataapiv1.StrictHTTPServerOptions{
```

Comment:
We may want to have this simply return Status Code 500 with no error body (Internal Server Error effectively) and file a bug to have improve on this.  Currently this means that it will be returning an error in a format that doesn't match the rest of the system, and potentially can leak internal details about the service.

Reply:
Done and I raised: https://jira.issues.couchbase.com/browse/ING-1017